mailbox/api/src/test/java/org/apache/james/mailbox/MailboxManagerTest.java

```diff
@@ -711,8 +712,11 @@ void setUp() throws Exception {
             newPath = MailboxPath.forUser(USER_1, "specialMailbox");
 
             listener = new EventCollector();
+            groupListener = new EventCollector();
             inboxId = mailboxManager.createMailbox(inbox, session).get();
             inboxManager = mailboxManager.getMailbox(inbox, session);
+
+            retrieveEventBus(mailboxManager).register(groupListener);
```

Comment:
and it's now registered also for tests that don't need it

Reply:
Is it an issue?

Reply:
yes it is. A test is supposed to have the minimal context to demonstrates something.

Reply:
All this group of test is specifically about events? I would consider this acceptable.